WordPress/gutenberg
# Changelog Automation: create top-level section for Accessibility PRs.

## What?

This PR updates the changelog generation script to create a top-level `Accessibility` section in the changelog for Accessibility PRs.

## Why?
So far, A11y PRs are miscatalogued in subsections, requiring release leads to manually move a11y items to their own top-level section.

## How?
Mapping the existing a11y-related labels to a new `Accessibility` top-level section.

## Testing Instructions
1. Run `npm run other:changelog -- --milestone="Gutenberg 15.4"`
2. Verify that a11y-related PRs appear in an `Accessibility` top-level section.
